specification/quantumleap.yml

```diff
@@ -280,6 +280,38 @@ parameters:
     example, if the query was to return 10 results and you use an offset of
     1, the response will return the last 9 values. Make sure you don't give
     more offset than the number of results."
+  georel:
+    in: query
+    name: georel
+    type: string
+    pattern: '^coveredBy$|^intersects$|^equals$|^disjoint$|^near;maxDistance:(0|([1-9][0-9]*))(\.[0-9]+)?$|^near;minDistance:(0|([1-9][0-9]*))(\.[0-9]+)?$|^near;maxDistance:(0|([1-9][0-9]*))(\.[0-9]+)?;minDistance:(0|([1-9][0-9]*))(\.[0-9]+)?$|^near;minDistance:(0|([1-9][0-9]*))(\.[0-9]+)?;maxDistance:(0|([1-9][0-9]*))(\.[0-9]+)?$'
+    description: "Optional. It specifies a spatial relationship between matching
+    entities and a reference shape (geometry). This parameter is used to perform
+    geographical queries with the same semantics as in the FIWARE-NGSI v2 Specification.
+    Full details can be found in the Geographical Queries section of the specification:
+    http://fiware.github.io/specifications/ngsiv2/stable/."
+  geometry:
+    in: query
+    name: geometry
+    type: string
+    enum: [point, line, polygon, box]
+    description: "Optional but required if georel is specified. This parameter
+    defines the reference shape to be used for geographical queries and has the
+    same semantics as in the FIWARE-NGSI v2 Specification.
+    Full details can be found in the Geographical Queries section of the specification:
+    http://fiware.github.io/specifications/ngsiv2/stable/."
+  coords:
+    in: query
+    name: coords
+    type: string
+    pattern: '^[+,-]?(0|([1-9][0-9]*))(\.[0-9]+)?,[+,-]?(0|([1-9][0-9]*))(\.[0-9]+)?(;[+,-]?(0|([1-9][0-9]*))(\.[0-9]+)?,[+,-]?(0|([1-9][0-9]*))(\.[0-9]+)?)*$'
+    description: "Optional but required if georel is specified. This parameter
```

Comment:
Yeah, we could. I was trying to avoid duplication since that stuff is already explained in the NSGI spec, so in the end I thought adding this would do: "same semantics as in the FIWARE-NGSI v2". In other words, this says: "we implement the NSGI v2 interface for geo queries, go look there for the details."

Do you think it's enough? Or should we actually copy over part of the description from the NSGI spec?

Reply:
my comment was supposed to go here, but email reply works badly apparently.
I would not report the old spec. I would provide the link.

Reply:
Oh yeap sure, that's enough.